src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala

```diff
@@ -198,7 +207,7 @@ object AppConfig {
       termsOfServiceConfig = config.as[TermsOfServiceConfig]("termsOfService"),
       oidcConfig = config.as[OidcConfig]("oidc"),
       adminConfig = config.as[AdminConfig]("admin"),
-      azureServicesConfig = config.getAs[AzureServicesConfig]("azureServices"),
+      azureServicesConfig = config.as[Option[AzureServicesConfig]]("azureServices"),
```

Comment:
Just trying to understand -- isn't `getAs[A]` and `as[Option[A]]` the same thing? Both return an `Option[A]`. Not sure I see why this has any effect..